### PR TITLE
grpc multirepo example

### PR DESCRIPTION
**Pull request by alexcb**

### CODE REVIEW

File: Earthfile [hunk 1]
```diff
@@ -211,23 +211,24 @@ test-all:
...
-    BUILD ./examples/js+docker
     BUILD ./examples/cpp+docker
-    BUILD ./examples/scala+docker
     BUILD ./examples/dotnet+docker
-    BUILD ./examples/python+docker
-    BUILD ./examples/ruby+docker
-    BUILD ./examples/ruby-on-rails+docker
     BUILD ./examples/elixir+docker
+    BUILD ./examples/go+docker
+    BUILD ./examples/grpc+test
```
Comment (alexcb) [inline Earthfile:218]: this is the only addition, the rest of the change is just sorting these alphabetically.

File: examples/grpc/Earthfile [hunk 2]
```diff
@@ -0,0 +1,39 @@
+FROM node:13.10.1-alpine3.11
+
+WORKDIR /example-grpc
+
+# This example shows how to use earthly to compile a protobuf grpc definition
+# into protobuf code for both a Go-based server, and a python-based client.
+#
+# This example spawns 3 different git repos:
```
Comment (vladaionescu) [inline examples/grpc/Earthfile:8]:
```suggestion
# This example spans 4 different git repos:
```

(The current file is in the 4th repo - might be clearer to the reader)

**Review by vladaionescu — APPROVED** [review]
:sunglasses:

File: examples/grpc/Earthfile (hunk 2 above)
Comment (alexcb) [reply examples/grpc/Earthfile]: lol, I must have been thinking about fish. I fixed this and slightly reworded the last 4th numbered item.

**Review by vladaionescu — APPROVED** [review]

**Pull request merged**